src/SDKs/SignalR/Management.SignalR/Microsoft.Azure.Management.SignalR.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.Management.SignalR</PackageId>
```

Comment:
nit: Please verify this is how the RP wants their packages to be named

Reply:
Done.